controller/static/bower.json

```diff
@@ -21,6 +21,7 @@
     "angular-resource": "~1.3.15",
     "angular-jwt": "~0.0.6",
     "jquery-tablesort": "*",
-    "angular-sanitize": "~1.3.15"
+    "angular-sanitize": "~1.3.15",
+    "angular-semantic-ui": "~0.0.2"
```

Comment:
Plus.. the semantic-ui dropdown javascript doesn't work properly when dynamically adding more `select` fields to the view.